operation/manager_test.go

```diff
@@ -11,155 +11,167 @@ import (
 	"go.viam.com/test"
 )
 
-func TestSingleOperationManager(t *testing.T) {
-	ctx := context.Background()
-	som := SingleOperationManager{}
+var (
+	ctx = context.Background()
```

Comment:
this context should be within tests. it's better practice to not have these two globals

Reply:
Okay, I'll move them back. Why is is better practice not to have them global?

Reply:
Expanding the answer I gave below: if they're global and reused in lots of tests, a failure in test A can leave it in a bad state, causing a failure in test B. and now you're stuck trying to figure out what's wrong with test B, and the answer is nothing at all, that the state was corrupted by test A, and it's really hard to figure that out because B is one of the tests that's failing so surely there's something wrong with B. 

By not reusing state between tests, you keep them independent of each other so that the failures are limited to the tests that actually have problems.

Reply:
so context is probably fine as a global since it's immutable but context in general is to be used on a request by request basis (it's request/api context)
op manager is very dangerous as global because of what Alan mentioned, shared state